src/main/java/bio/terra/workspace/service/workspace/flight/CreateGoogleContextRBSFlight.java

```diff
@@ -9,11 +9,14 @@
 
 // NOTE: DO NOT USE. Currently just a shell class to exercise connection to Buffer Service.
 public class CreateGoogleContextRBSFlight extends Flight {
+  // Buffer Retry rule settings. For Buffer Service, allow for long wait times.
+  // If the pool is empty, RBS may need time to actually create a new project.
+  private static final int BUFFER_INITIAL_INTERVAL_SECONDS = 1;
+  private static final int BUFFER_MAX_INTERVAL_SECONDS = 5 * 60;
+  private static final int BUFFER_MAX_OPERATION_TIME_SECONDS = 60 * 15;
```

Comment:
optional: consider making the bufferRetryRule `private static final` instead of its constants.

Reply:
This was the pattern Dan left when he refactored, so I think I will just leave it.